meetings/2020-06-22.md

```diff
@@ -1,17 +1,100 @@
-     So one VFX house would have something prepared for the next meeting,
-     and we round robin.
+# 22 June 2020
+
+Presents
+- Ashley Whetter
+- Ryan Bottriedll
+- JT Nelson
+- Ricky Rieckenberg
+- Mark Wiebe
+- John Mertic
+- Ollie Harding 
+- Dora Morolli
+- Zhicheng Ye
+- David Aguilar
+
+## Election of a permanent chair
+- Ashley: we will leave this for another meeting, when there are more people present
+
+## How to get vendors involved
+- Ashley: it was originally the vendors idea to set up this group. I'm working with Daniel on it. Any idea how to get them involved?
+- Ollie: we should not wait too long for formulate a plan - it is coming!
+
+## Creation of deliverables
+- Ashley: How do we want to go about creating some of the documents we mentioned last time? Best practices of transitions, common problems, maintainable list of practices..
+- We spoke about a document updated regularly, via Google Drive or GitHub, rather than having it updated in this meeting
+- How do we want the document happen? i..e in the background vs in this meeting
+- Dora: it's preferable to use GitHub as it's more open and visible
+- Ricky: we can use the Slack channel for discussion
+- Ashley: Slack can be used for instant discussion, as well as using this meeting for having quick resolves
+- Ryan: if it's static documentation it makes sense in GitHub, but maybe something more collaborative would be easier (as in Google Docs)
+- Ashley: we can link to the Google Docs in GitHub. Notification is a problem so we could use this meeting to outline what has changed
+- Ryan: how does the Linux foundation do this kind of things?
+- John: we can set up something like this with our mailing list. Also GitHub has a new discussion feature that some communities start to use. There are a lot of different options, depending on the engagement we want. The mailing list should be already subscribable and most of our communities use a GitHub repo with a Contributing.md and a Readme.md, how to get involved (similar to a code project)
+- Ryan: I feel GitHub might get in the way because of the authentication issues, but we could try to stick with GH and see how it goes
+
+## Current blockers
+- Ollie: we are blocked by not knowing dates
+- Mark: there are some things that we can do like import future. Are people doing that?
+- Ryan: we should futurizing our code but there's no motivation as we don't have any dates
+- Ashley: it's a big challenge and it's going to take a lot of development time. we want to spread it 
+- Ricky: similar, we want to spread as much as possible 
+- Ashley: how? we have been thinking to use a tool called "python modernise" 
+- Ricky: we are still looking at it with python linters, 2to3 reports
+- Ryan: we use pylint with the -3k flag and capturing what numbers of issues we have in our repos
+- Ashley: we have looked that pylint reports look practically identical. I'm curious to see how many false positives we are going to have
```

Comment:
To clarify this point, I mentioned that the pylint reports are almost identical to the `python-modernize` ones.